drivers/sensor/pms7003/pms7003.c

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* sensor pms7003.c - Driver for plantower PMS7003 sensor
+ * PMS7003 product: http://www.plantower.com/en/content/?110.html
+ * PMS7003 spec: http://aqicn.org/air/view/sensor/spec/pms7003.pdf
```

Comment:
I'm guessing there is not spec/datasheet in English, right?

Reply:
yes,this is a Chinese version